source/_references/10-operations.md

```diff
@@ -164,6 +164,18 @@ Adds a dropshadow to an image.
 
 Converts an image to grayscale.
 
+### Modulate
+
+Modulates an image for Brightness, Saturation and Hue. Use this to for example brighten up or darken an image or 
+remove its colors. The value is meant as percentage, with which the different values are multiplied. 
+100 leaves the property untouched, 0 removes all of it. 
+
+#### Properties
+
+- `brightness`: Integer, between 0 and 500. The default value is 100.
+- `saturation`: Integer, between 0 and 500. The default value is 100.
+- `hue`: Integer, between 0 and 500. The default value is 100.
```

Comment:
is hue a percentage? isn't hue kindof a circle on the color wheel?

Reply:
It basically does this:
http://www.imagemagick.org/Usage/color_mods/#modulate
And I'm not sure about the hue part, I guess we do it differently (since it's vips and not imagemagick), I have to actually check